CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 1.0.1 (2023-11-07)
+
```

Comment:
Strange... CHANGELOG is empty.
I think you didnt "Squash and merge" the previous PRs, so the conventional commits aren't in the history if you look at history on master.

Note: This means we dont _need_ one or more conventional commits to. make a release, looks like commitizen will bump the patch in the absence of any conventional commits - its just the CHANGELOG that suffers.

cc @semirp

Reply:
Maybe I just merged 🤦. 
Note to self,  SquashAndMerge next time.

I updated the changelog manually. @NQuinn27